domain/servicefactory/testing/suite.go

```diff
@@ -76,13 +76,13 @@ func (s stubDBDeleter) DeleteDB(namespace string) error {
 // ControllerServiceFactory conveniently constructs a service factory for the
 // controller model.
 func (s *ServiceFactorySuite) ControllerServiceFactory(c *gc.C) servicefactory.ServiceFactory {
-	return s.ServiceFactoryGetter(c)(string(s.ControllerModelUUID))
+	return s.ServiceFactoryGetter(c)(s.ControllerModelUUID)
 }
```

Comment:
The name of this method is bad. This should be `ControllerModelServiceFactory`.

Reply:
it has 169 usages, do you want me to change them all over in this PR?

Reply:
I think follow up PR if we want to change this. This is already in the code. Quick fixes are one thing but this would pollute this PR.

Reply:
@tlm agree